README.md

```diff
@@ -198,6 +198,7 @@ var signers = [
         email_address : 'jack@example.com',
         name : 'Jack',
         order : 0,
+        sms_phone_number: '+14155552671',
```

Comment:
Technically, only `555-0100` through `555-0199` are reserved for fictional use; the other numbers have been reserved for actual assignment*. Might be better to choose a number within that range instead

Reply:
Updated to reflect your suggestion.